src/libraries/System.Diagnostics.Process/src/System/Diagnostics/PerformanceCounterLib.cs

```diff
@@ -57,15 +57,15 @@ internal PerformanceCounterLib(string machineName, string lcid)
 
         internal string GetCounterName(int index)
         {
-            string result;
+            string? result;
             return NameTable.TryGetValue(index, out result) ? result : "";
         }
 
         internal static PerformanceCounterLib GetPerformanceCounterLib(string machineName, CultureInfo culture)
         {
             string lcidString = culture.Name.ToLowerInvariant();
             if (machineName.CompareTo(".") == 0)
-                machineName = ComputerName.ToLowerInvariant();
+                machineName = ComputerName!.ToLowerInvariant();
```

Comment:
I'm confused.  Can or can't ComputeName return null?

Reply:
Strictly speaking, it might be possible to return null https://github.com/dotnet/runtime/pull/31671#discussion_r375377710

Reply:
If it can return null, then this is a bug we should fix.

I'm inclined, however, to say we should address the various issues I cited by doing `GetComputerName() ?? ""` as part of the lazy initialization.

Reply:
Done.